app/src/main/java/com/cornellappdev/score/components/SportSelectorHeader.kt

```diff
@@ -0,0 +1,161 @@
+    ) {
+        Row(
+            modifier = Modifier
+                .padding(end = 24.dp)
+                .border(
+                    border = BorderStroke(1.dp, GrayLight),
+                    shape = RoundedCornerShape(100)
+                )
+                .padding(6.dp)
+        ) {
+            GenderSelector("All", selectedOption) {
+                selectedOption = "All"
+                onGenderSelected("All")
+            }
+            GenderSelector("Men's", selectedOption) {
+                selectedOption = "Men's"
+                onGenderSelected("Men's")
+            }
+            GenderSelector("Women's", selectedOption) {
+                selectedOption = "Women's"
+                onGenderSelected("Women's")
+            }
+        }
+        LazyRow(Modifier.fillMaxWidth()) {
+            items(sports) { (sport, emptyIcon, filledIcon) ->
+                SportSelector(
+                    empty = emptyIcon,
+                    filled = filledIcon,
+                    option = sport,
+                    selectedOption = selectedSport
+                ) {
+                    selectedSport = sport
+                    onSportSelected(sport)
+                }
+
+                Spacer(modifier = Modifier.width(20.dp))
+            }
+            item { Spacer(modifier = Modifier.width(4.dp)) }
+        }
+    }
+}
+
+@Composable
+fun GenderSelector(option: String, selectedOption: String, onSelect: () -> Unit) {
+    Box(
+        modifier = Modifier
+            .clickable(onClick = {
+                onSelect()
+            })
+            .width(111.dp)
+            .background(
+                color = if (selectedOption == option) CrimsonPrimary else Color.Transparent,
+                shape = RoundedCornerShape(100) // Rounded background
+            )
+            .padding(vertical = 8.dp, horizontal = 12.dp),
+        contentAlignment = Alignment.Center
+    ) {
+        Text(
+            text = option,
+            style = if (selectedOption == option) genderFilterText else genderText
+        )
+    }
+}
+
+@Composable
+fun SportSelector(
+    empty: Painter,
+    filled: Painter,
+    option: String,
+    selectedOption: String,
+    onSelect: () -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .clickable(onClick = {
+                onSelect()
+            })
+            .sizeIn(minWidth = 56.dp),
+        verticalArrangement = Arrangement.spacedBy(6.dp),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Image(
+            painter = if (selectedOption == option) filled else empty,
+            contentDescription = "Sport Icon",
+            modifier = Modifier
+                .padding(1.dp)
+                .width(24.dp)
+                .height(24.dp)
+        )
+        Text(
+            text = option,
+            style = sportFilterText,
+            color = if (selectedOption == option) CrimsonPrimary else GrayMedium
+        )
+    }
+}
```

Comment:
Huh, this stuff shouldn't be in this PR right? This was in your other PR.

Reply:
I guess you can wait for the other to be merged, then rebase on main. The diff here should be cleaned up afterwards. Or, ideally, more work, but you can just manually pull out the changes here and put them in a new branch branched from `main`.

Reply:
To be clear, it seems like you made this branch off your other branch instead of `main`.

Reply:
I think this is just related to my Git/GitHub issues because I'm quite confident that I created this branch off of `main` :')